public/app/routes/routes.ts

```diff
@@ -436,9 +437,10 @@ export function setupAngularRoutes($routeProvider: route.IRouteProvider, $locati
       },
     })
     .when('/alerting/notifications', {
-      templateUrl: 'public/app/features/alerting/partials/notifications_list.html',
-      controller: 'AlertNotificationsListCtrl',
-      controllerAs: 'ctrl',
+      template: '<react-container />',
+      resolve: {
+        component: () => NotificationsListPage,
```

Comment:
Maybe use SafeDynamicImport?
```suggestion
        component: () => SafeDynamicImport(import(/* webpackChunkName: "NotificationsListPage" */ '...../.NotificationsListPage')),,
```

Reply:
I did, but the page had issues loading